packages/scheduler/scheduler.py

```
@@ -348,13 +348,13 @@ async def _ensure_store(self) -> None:
         # tests that inject via mod._store) and fall back to the agent-level
         # ScheduleStore when it isn't available.
         try:
-            from services.scheduler_store import get_scheduler_store
+            from packages.scheduler.store import get_scheduler_store
             self._store = get_scheduler_store()
             return
         except Exception:
             pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)